task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,30 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    //throw new Error('Not implemented');
+    let str = '';
+	
+	/*if(num % 3 === 0 && num % 5 === 0){
+        str = 'FizzBuzz';
+    }
+    else if(num % 3 === 0){
+        str = 'Fizz';
+    }
+    else if(num % 5 === 0){
+        str = 'Buzz';
+    }
+    else str = num;*/
+
+    if(num % 3 === 0){
+        str = 'Fizz';
+    }
+	if(num % 5 === 0){
```

Comment:
Must be indented after conditional statements and before bracket.
https://javascript.info/coding-style